### DISCUSSION TITLE
Save model as pkl file

### DISCUSSION THREAD

**Discussion by solarflarefx** [facebookresearch/detectron2#3535] [Q&A]
I was experimenting with this package: https://github.com/tensorpack/tensorpack/tree/master/examples/FasterRCNN/convert_d2

It looks like it requires saving the detectron2 model weights in .pkl file.  Right now I have the weights saved at .pth.  Is there an easy way to save as .pkl?

**Comment by yangdesheng** [comment]
model = Trainer.build_model(cfg)
DetectionCheckpointer(model).resume_or_load(cfg.MODEL.WEIGHTS, resume=False )#args.resume
checkpointer=DetectionCheckpointer(model, save_dir=cfg.OUTPUT_DIR)
with open(os.path.join(cfg.OUTPUT_DIR,"model_999.pkl"),'wb') as f:
    pickle.dump(checkpointer,f)

**Reply by ngadhvi** [reply]
Did this work? @solarflarefx
